suite-native/atoms/src/Loader.tsx

```diff
@@ -0,0 +1,31 @@
+import { ActivityIndicator } from 'react-native';
+import React from 'react';
+
+import { useNativeStyles } from '@trezor/styles/';
+import { Color } from '@trezor/theme/';
+
+import { Box } from './Box';
+import { Text } from './Text';
+
+type LoaderProps = {
+    title?: string;
+    color?: Color;
+};
+
+// TODO: modify component to fit Figma design.
+// issue: https://github.com/trezor/trezor-suite/issues/7538
```

Comment:
No need to write todo comments if issues are created. Please delete. Just add the issue to sprint when it should be done.

Reply:
In one of my previous issues, @Nodonisko asked me to attach a follow-up issue like this. Please let's agree all together if it makes sense to do it or not.

Reply:
I don't know which one you're mentioning so don't know the context. But since issue is created there is no need to pollute code with TODO's referencing these issues.

Reply:
Only found this so I guess it's decided. 😅
<img width="746" alt="image" src="https://user-images.githubusercontent.com/36101761/217199217-58a0721a-20a4-41a0-8e09-af780036b176.png">

Reply:
this is not the case

Reply:
Send link? 

Anyway it's my opinion this is not necessary. If @Nodonisko wants to enforce it I'm okay with that but it doesn't make sense to pollute code with issue links when the point of issue is not having to keep your TODO's elsewhere. If issue gets deleted due to priorities etc, you have dead TODO here that somebody will have to delete later.

Reply:
https://github.com/trezor/trezor-suite/pull/7454#discussion_r1087927395

Reply:
Hmm. Let @Nodonisko decide, it's just a nit as well.

Reply:
I don't have any strong rule about this. I would say if there is issue it's usually not necessary to have comment in code. Exception is when code is somehow complicated so it may not be clear from first sight it's related. In this case I think you can't miss this component so comment is not necessary, but I don't mind if you want to keep it here.

If I should throw in example when it makes sense to put comment, it could be for example some places where is old ActivityIndicator still used and should be removed after this component will be refactored. Then comment make sense because it's not directly related and may be forgotten.